platform/ios/src/MGLMapView.h

```diff
@@ -1300,6 +1300,34 @@ IB_DESIGNABLE
  */
 - (NS_ARRAY_OF(id <MGLFeature>) *)visibleFeaturesInRect:(CGRect)rect inStyleLayersWithIdentifiers:(nullable NS_SET_OF(NSString *) *)styleLayerIdentifiers predicate:(nullable NSPredicate *)predicate NS_SWIFT_NAME(visibleFeatures(in:styleLayerIdentifiers:predicate:));
 
+/**
+ Returns an array of map features for the given source, restricted to the given 
+ source layer and filtered by the given predicate.
+ 
+ Each object in the returned array represents a feature for the
+ current style and provides access to attributes specified by the relevant
+ <a href="https://www.mapbox.com/mapbox-gl-style-spec/#sources">tile sources</a>.
+ The returned array includes features specified in vector and GeoJSON tile
+ sources but does not include anything from raster, image, or video sources.
+
+ Features come from tiled vector data or GeoJSON data that is converted to tiles
+ internally, so feature geometries are clipped at tile boundaries and features
+ may appear duplicated across tiles. For example, suppose the specified
+ rectangle intersects with a road that spans the screen. The resulting array
```

Comment:
Some of this explanation is specific to -visibleFeaturesInRect:. This method by contrast accepts no rectangle parameter.